application/views/_playerinfo1.php

```diff
@@ -0,0 +1,9 @@
+
+<table>
```

Comment:
@jotilalli The <table> tags should be outside this view otherwise we will have multiple tables when this iterates through.